src/vm/excep.cpp

```diff
@@ -5152,10 +5138,28 @@ LONG InternalUnhandledExceptionFilter(
 
 } // LONG InternalUnhandledExceptionFilter()
 
+bool& GetUseEntryPointFilter()
```

Comment:
This is method looks weird. Can this be just a simple static variable and the ifdef moved to `ParseUseEntryPointFilter`?

Reply:
Sure, but I thought you didn't like simple global static variables.  One of my earlier reviews there was concern with initialization startup races for global static variables.

Reply:
We have hundreds of simple global static variables. I do not see a problem with them.

> concern with initialization startup races for global static variables

This must have been something slightly different. Maybe global static variables with destructors?